packages/nimble-components/src/anchor/tests/anchor.spec.ts

```diff
@@ -80,6 +80,25 @@ describe('Anchor', () => {
                 expect(element.control!.getAttribute(name)).toBe('foo');
             });
         });
+
+        it('for attribute tabindex', async () => {
+            await connect();
+
+            element.setAttribute('tabindex', '-1');
+            await waitForUpdatesAsync();
+
+            expect(element.control!.getAttribute('tabindex')).toBe('-1');
+        });
+    });
+
+    it('should clear tabindex attribute from the internal control when changed to null', async () => {
+        await connect();
+
+        element.setAttribute('tabindex', '-1');
+        element.setAttribute('tabindex', 'null');
```

Comment:
Setting the attribute to `'null'` for the test (the actual string `'null'`) seems strange to me (I don't think any code paths should do that in practice)

Verifying that we can clear tabIndex seems like a valid test, but that would just be `element.removeAttribute('tabindex')` instead, for this line.

(Same comment applies to the other control tests)

Reply:
That test was meant to verify that we are using the `nullableNumberConverter` for `tabindex`. I'm testing that if a binding ends up evaluating to `null`, it removes the attribute. I think `setAttribute('tabindex', 'null')` should be equivalent to a template with `tabindex="${x => x.foo}"` where `x.foo` evaluates to `null`. Is that wrong?

I suppose we could _also_ test that clearing the attribute with `removeAttribute('tabindex')` also reflects to the inner element.

Reply:
As best as I can tell (stepping through the current checkbox binding code without these changes), `setAttribute('tabindex', 'null')` wouldn't be what happens. Since in templates we're generally binding to `tabindex` as an attribute, we go through FAST's [updateAttributeTarget()](https://github.com/microsoft/fast/blob/9c6dbb66615e6d229fc0ebf8065a67f109139f26/packages/web-components/fast-element/src/templating/binding.ts#L72), and then their [DOM.setAttribute()](https://github.com/microsoft/fast/blob/9c6dbb66615e6d229fc0ebf8065a67f109139f26/packages/web-components/fast-element/src/dom.ts#L208), which calls `removeAttribute()` on the element for null/undefined values.

I actually don't think we need the 'nullable' part of the converter, but I didn't see a plain number converter without that to use (without making one). In any case, I think the tests that do `removeAttribute()` are sufficient for how we'll be using tabindex.